src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/GameEmbedProjection.java

```diff
@@ -0,0 +1,19 @@
+package ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity;
+
+import java.util.Collection;
+import org.springframework.data.rest.core.config.Projection;
+
+@Projection(
+    name = "embed",
+    types = {Game.class})
+public interface GameEmbedProjection {
+  Long getId();
+
+  String getName();
+
+  GameState getGameState();
+
+  Collection<ParticipationEmbedProjection> getParticipations();
```

Comment:
With the projection, we can directly embed the related entity, and even use their projection.